### ISSUE TITLE
hbase - 2020-07-03 02:56:51,620 ERROR [main] zookeeper.ZooKeeperWatcher: hconnection-0x5e5efbee0x0, quorum=hdp-node-01:2181,hdp-node-02:2181,hdp-node-03:2181, baseZNode=/hbase Received unexpected KeeperException, re-throwing exception org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid

### ISSUE THREAD

**Issue by gitava** [gitava/Big-Data-Study#33]
after using start-hbase.sh w/o sudo, hbase shell report the following errors ...
```
[vagrant@hdp-node-01 ~/hbase/logs]$~/hbase/bin/start-hbase.sh
hdp-node-03: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-03.out
hdp-node-02: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-02.out
hdp-node-01: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-01.out
running master, logging to /home/vagrant/hbase/logs/hbase-vagrant-master-hdp-node-01.out
hdp-node-02: regionserver running as process 4194. Stop it first.
hdp-node-03: regionserver running as process 4182. Stop it first.
hdp-node-01: regionserver running as process 8830. Stop it first.
[vagrant@hdp-node-01 ~/hbase/logs]$kill -9 4194
-bash: kill: (4194) - No such process
[vagrant@hdp-node-01 ~/hbase/logs]$kill -9 8830
-bash: kill: (8830) - No such process
[vagrant@hdp-node-01 ~/hbase/logs]$~/hbase/bin/start-hbase.sh
hdp-node-03: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-03.out
hdp-node-02: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-02.out
hdp-node-01: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-01.out
running master, logging to /home/vagrant/hbase/logs/hbase-vagrant-master-hdp-node-01.out
hdp-node-02: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-02.out
hdp-node-03: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-03.out
hdp-node-01: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-01.out
[vagrant@hdp-node-01 ~/hbase/logs]$hbase shell
2020-07-03 02:56:51,616 ERROR [main] zookeeper.RecoverableZooKeeper: ZooKeeper exists failed after 4 attempts
2020-07-03 02:56:51,618 WARN  [main] zookeeper.ZKUtil: hconnection-0x5e5efbee0x0, quorum=hdp-node-01:2181,hdp-node-02:2181,hdp-node-03:2181, baseZNode=/hbase Unable to set watcher on znode (/hbase/hbaseid)
org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:99)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:1102)
	at org.apache.hadoop.hbase.zookeeper.RecoverableZooKeeper.exists(RecoverableZooKeeper.java:284)
	at org.apache.hadoop.hbase.zookeeper.ZKUtil.checkExists(ZKUtil.java:468)
	at org.apache.hadoop.hbase.zookeeper.ZKClusterId.readClusterIdZNode(ZKClusterId.java:65)
	at org.apache.hadoop.hbase.client.ZooKeeperRegistry.getClusterId(ZooKeeperRegistry.java:105)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.retrieveClusterId(ConnectionManager.java:944)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.<init>(ConnectionManager.java:720)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.<init>(ConnectionManager.java:649)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:238)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:218)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:450)
	at org.jruby.javasupport.JavaMethod.invokeStaticDirect(JavaMethod.java:362)
	at org.jruby.java.invokers.StaticMethodInvoker.call(StaticMethodInvoker.java:58)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:312)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:169)
	at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
	at org.jruby.ast.InstAsgnNode.interpret(InstAsgnNode.java:95)
	at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:104)
	at org.jruby.ast.BlockNode.interpret(BlockNode.java:71)
	at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
	at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:169)
	at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:191)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:302)
	at org.jruby.runtime.callsite.CachingCallSite.callBlock(CachingCallSite.java:144)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:148)
	at org.jruby.RubyClass.newInstance(RubyClass.java:822)
	at org.jruby.RubyClass$i$newInstance.call(RubyClass$i$newInstance.gen:65535)
	at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodZeroOrNBlock.call(JavaMethod.java:249)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:292)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:135)
	at home.vagrant.hbase.bin.hirb.__file__(/home/vagrant/hbase/bin/hirb.rb:142)
	at home.vagrant.hbase.bin.hirb.load(/home/vagrant/hbase/bin/hirb.rb)
	at org.jruby.Ruby.runScript(Ruby.java:697)
	at org.jruby.Ruby.runScript(Ruby.java:690)
	at org.jruby.Ruby.runNormally(Ruby.java:597)
	at org.jruby.Ruby.runFromMain(Ruby.java:446)
	at org.jruby.Main.doRunFromMain(Main.java:369)
	at org.jruby.Main.internalRun(Main.java:258)
	at org.jruby.Main.run(Main.java:224)
	at org.jruby.Main.run(Main.java:208)
	at org.jruby.Main.main(Main.java:188)
2020-07-03 02:56:51,620 ERROR [main] zookeeper.ZooKeeperWatcher: hconnection-0x5e5efbee0x0, quorum=hdp-node-01:2181,hdp-node-02:2181,hdp-node-03:2181, baseZNode=/hbase Received unexpected KeeperException, re-throwing exception
org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:99)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:1102)
	at org.apache.hadoop.hbase.zookeeper.RecoverableZooKeeper.exists(RecoverableZooKeeper.java:284)
	at org.apache.hadoop.hbase.zookeeper.ZKUtil.checkExists(ZKUtil.java:468)
	at org.apache.hadoop.hbase.zookeeper.ZKClusterId.readClusterIdZNode(ZKClusterId.java:65)
	at org.apache.hadoop.hbase.client.ZooKeeperRegistry.getClusterId(ZooKeeperRegistry.java:105)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.retrieveClusterId(ConnectionManager.java:944)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.<init>(ConnectionManager.java:720)
	at org.apache.hadoop.hbase.client.ConnectionManager$HConnectionImplementation.<init>(ConnectionManager.java:649)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:238)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:218)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:450)
	at org.jruby.javasupport.JavaMethod.invokeStaticDirect(JavaMethod.java:362)
	at org.jruby.java.invokers.StaticMethodInvoker.call(StaticMethodInvoker.java:58)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:312)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:169)
	at org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
	at org.jruby.ast.InstAsgnNode.interpret(InstAsgnNode.java:95)
	at org.jruby.ast.NewlineNode.interpret(NewlineNode.java:104)
	at org.jruby.ast.BlockNode.interpret(BlockNode.java:71)
	at org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
	at org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.java:169)
	at org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.java:191)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:302)
	at org.jruby.runtime.callsite.CachingCallSite.callBlock(CachingCallSite.java:144)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:148)
	at org.jruby.RubyClass.newInstance(RubyClass.java:822)
	at org.jruby.RubyClass$i$newInstance.call(RubyClass$i$newInstance.gen:65535)
	at org.jruby.internal.runtime.methods.JavaMethod$JavaMethodZeroOrNBlock.call(JavaMethod.java:249)
	at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:292)
	at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:135)
	at home.vagrant.hbase.bin.hirb.__file__(/home/vagrant/hbase/bin/hirb.rb:142)
	at home.vagrant.hbase.bin.hirb.load(/home/vagrant/hbase/bin/hirb.rb)
	at org.jruby.Ruby.runScript(Ruby.java:697)
	at org.jruby.Ruby.runScript(Ruby.java:690)
	at org.jruby.Ruby.runNormally(Ruby.java:597)
	at org.jruby.Ruby.runFromMain(Ruby.java:446)
	at org.jruby.Main.doRunFromMain(Main.java:369)
	at org.jruby.Main.internalRun(Main.java:258)
	at org.jruby.Main.run(Main.java:224)
	at org.jruby.Main.run(Main.java:208)
	at org.jruby.Main.main(Main.java:188)
HBase Shell
Use "help" to get list of supported commands.
Use "exit" to quit this interactive shell.
Version 1.6.0, r5ec5a5b115ee36fb28903667c008218abd21b3f5, Fri Feb 14 12:00:03 PST 2020
```

**Comment by gitava** [comment]
try deleting all zookeeper contents and try again...

**Comment by gitava** [comment]
hbase files on hdfs are all roots..change that first...
```sh
[vagrant@hdp-node-01 ~/hbase/logs]$hdfs dfs -ls /hbase
Found 7 items
drwxr-xr-x   - root supergroup          0 2020-07-03 02:45 /hbase/.tmp
drwxr-xr-x   - root supergroup          0 2020-07-03 02:45 /hbase/MasterProcWALs
drwxr-xr-x   - root supergroup          0 2020-07-03 02:46 /hbase/WALs
drwxr-xr-x   - root supergroup          0 2020-07-03 02:27 /hbase/data
-rw-r--r--   3 root supergroup         42 2020-07-03 02:27 /hbase/hbase.id
-rw-r--r--   3 root supergroup          7 2020-07-03 02:27 /hbase/hbase.version
drwxr-xr-x   - root supergroup          0 2020-07-03 02:27 /hbase/oldWALs
[vagrant@hdp-node-01 ~/hbase/logs]$hdfs dfs -ls /
Found 7 items
drwxr-xr-x   - vagrant   supergroup          0 2020-07-03 02:45 /hbase
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:13 /input
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:33 /output
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 02:28 /temp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:09 /tmp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:40 /user
drwxr-xr-x   - vagrant   supergroup          0 2020-05-27 12:31 /wordcount
```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-01 ~/hbase/logs]$hdfs dfs -rm -r /hbase/*
Deleted /hbase/.tmp
Deleted /hbase/MasterProcWALs
Deleted /hbase/WALs
Deleted /hbase/data
Deleted /hbase/hbase.id
Deleted /hbase/hbase.version
Deleted /hbase/oldWALs
[vagrant@hdp-node-01 ~/hbase/logs]$hdfs dfs -ls /
Found 7 items
drwxr-xr-x   - vagrant   supergroup          0 2020-07-03 03:02 /hbase
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:13 /input
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:33 /output
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 02:28 /temp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:09 /tmp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:40 /user
drwxr-xr-x   - vagrant   supergroup          0 2020-05-27 12:31 /wordcount
[vagrant@hdp-node-01 ~/hbase/logs]$hdfs dfs -ls /hbase
[vagrant@hdp-node-01 ~/hbase/logs]$
```

**Comment by gitava** [comment]
one 3 servers ....

```
sudo rm -rf ~/hbase/zookeeper/*
```

**Comment by gitava** [comment]
```
vagrant@hdp-node-01 ~/hbase]$~/hbase/bin/start-hbase.sh
hdp-node-03: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-03.out
hdp-node-02: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-02.out
hdp-node-01: running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-01.out
running master, logging to /home/vagrant/hbase/logs/hbase-vagrant-master-hdp-node-01.out
hdp-node-02: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-02.out
hdp-node-03: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-03.out
hdp-node-01: running regionserver, logging to /home/vagrant/hbase/logs/hbase-vagrant-regionserver-hdp-node-01.out
[vagrant@hdp-node-01 ~/hbase]$sudo jps
4945 DataNode
11202 HQuorumPeer
5140 SecondaryNameNode
11541 Jps
11445 HRegionServer
11302 HMaster
4839 NameNode
2812 NodeManager
3038 JobHistoryServer
2687 ResourceManager
```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-02 ~/hbase]$sudo jps
2227 NodeManager
5188 Jps
5031 HRegionServer
2809 DataNode
4910 HQuorumPeer
```

```
[vagrant@hdp-node-03 ~/hbase/zookeeper]$sudo jps
5223 Jps
4895 -- process information unavailable
2227 NodeManager
5025 -- process information unavailable
2788 DataNode
```

**Comment by gitava** [comment]
got a new error

```
[vagrant@hdp-node-01 ~/hbase]$hbase shell
HBase Shell
Use "help" to get list of supported commands.
Use "exit" to quit this interactive shell.
Version 1.6.0, r5ec5a5b115ee36fb28903667c008218abd21b3f5, Fri Feb 14 12:00:03 PST 2020

hbase(main):001:0> list
TABLE

ERROR: Can't get master address from ZooKeeper; znode data == null

Here is some help for this command:
List all user tables in hbase. Optional regular expression parameter could
be used to filter the output. Examples:

  hbase> list
  hbase> list 'abc.*'
  hbase> list 'ns:abc.*'
  hbase> list 'ns:.*'


hbase(main):002:0>
```

**Comment by gitava** [comment]
under tmp,there are root stuff too...delete them on 3 servers

```
[vagrant@hdp-node-01 ~/hbase/tmp]$ls
local
[vagrant@hdp-node-01 ~/hbase/tmp]$ls -ltr
total 0
drwxr-xr-x. 3 root root 18 Jul  2 12:04 local
[vagrant@hdp-node-01 ~/hbase/tmp]$ls -ltrR
.:
total 0
drwxr-xr-x. 3 root root 18 Jul  2 12:04 local

./local:
total 0
drwxr-xr-x. 2 root root 6 Jul  2 12:04 jars

./local/jars:
total 0
[vagrant@hdp-node-01 ~/hbase/tmp]$sudo rm -rf ~/hbase/tmp/*
[vagrant@hdp-node-01 ~/hbase/tmp]$ll
total 0
```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-01 ~/hbase]$hdfs dfs -ls /
Found 7 items
drwxr-xr-x   - vagrant   supergroup          0 2020-07-03 03:04 /hbase
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:13 /input
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 03:33 /output
drwxr-xr-x   - fllbeaver supergroup          0 2020-06-24 02:28 /temp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:09 /tmp
drwxr-xr-x   - vagrant   supergroup          0 2020-06-28 06:40 /user
drwxr-xr-x   - vagrant   supergroup          0 2020-05-27 12:31 /wordcount
[vagrant@hdp-node-01 ~/hbase]$hdfs dfs -ls /hbase
Found 6 items
drwxr-xr-x   - vagrant supergroup          0 2020-07-03 03:04 /hbase/.tmp
drwxr-xr-x   - vagrant supergroup          0 2020-07-03 03:04 /hbase/MasterProcWALs
drwxr-xr-x   - vagrant supergroup          0 2020-07-03 03:04 /hbase/data
-rw-r--r--   3 vagrant supergroup         42 2020-07-03 03:04 /hbase/hbase.id
-rw-r--r--   3 vagrant supergroup          7 2020-07-03 03:04 /hbase/hbase.version
drwxr-xr-x   - vagrant supergroup          0 2020-07-03 03:04 /hbase/oldWALs
```

**Comment by gitava** [comment]
cant start zookeeper on hdp03

try manually starting it

```
hbase-daemon.sh start zookeeper 

```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-03 ~]$hbase-daemon.sh stop zookeeper
running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-03.out
stopping zookeeper.
[vagrant@hdp-node-03 ~]$hbase-daemon.sh start zookeeper
running zookeeper, logging to /home/vagrant/hbase/logs/hbase-vagrant-zookeeper-hdp-node-03.out
[vagrant@hdp-node-03 ~]$sudo jps
2855 Jps
2113 DataNode
2226 NodeManager
2534 -- process information unavailable
2775 -- process information unavailable
```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-01 ~]$hbase shell
HBase Shell
Use "help" to get list of supported commands.
Use "exit" to quit this interactive shell.
Version 1.6.0, r5ec5a5b115ee36fb28903667c008218abd21b3f5, Fri Feb 14 12:00:03 PST 2020

hbase(main):001:0> list
TABLE

ERROR: Can't get master address from ZooKeeper; znode data == null

Here is some help for this command:
List all user tables in hbase. Optional regular expression parameter could
be used to filter the output. Examples:

  hbase> list
  hbase> list 'abc.*'
  hbase> list 'ns:abc.*'
  hbase> list 'ns:.*'


hbase(main):002:0> exit
```

**Comment by gitava** [comment]
after fix of #35, this error just disappeared.
```
[vagrant@hdp-node-01 ~]$hbase shell
HBase Shell
Use "help" to get list of supported commands.
Use "exit" to quit this interactive shell.
Version 1.6.0, r5ec5a5b115ee36fb28903667c008218abd21b3f5, Fri Feb 14 12:00:03 PST 2020

hbase(main):001:0> list
TABLE
0 row(s) in 0.4230 seconds

=> []
hbase(main):002:0> create 't2',{NAME=>'f1'}
0 row(s) in 1.5680 seconds

=> Hbase::Table - t2
hbase(main):003:0> describe t2
NameError: undefined local variable or method `t2' for #<Object:0x4094b6c9>

hbase(main):004:0> describe 't2'
Table t2 is ENABLED
t2
COLUMN FAMILIES DESCRIPTION
{NAME => 'f1', DATA_BLOCK_ENCODING => 'NONE', BLOOMFILTER => 'ROW', REPLICATION_SCOPE => '0', VERSIONS => '1', COMPRESSION =>
 'NONE', MIN_VERSIONS => '0', TTL => 'FOREVER', KEEP_DELETED_CELLS => 'FALSE', BLOCKSIZE => '65536', IN_MEMORY => 'false', BL
OCKCACHE => 'true'}
1 row(s) in 0.0480 seconds

hbase(main):005:0>
```
